examples/aishell/s0/run.sh

```diff
@@ -1,32 +1,32 @@
 #!/bin/bash
 
 # Copyright 2019 Mobvoi Inc. All Rights Reserved.
-
 . ./path.sh || exit 1;
-. ./cmd.sh || exit 1;
 
 # Use this to control how many gpu you use, It's 1-gpu training if you specify
 # just 1gpu, otherwise it's is multiple gpu training based on DDP in pytorch
-export CUDA_VISIBLE_DEVICES="0,1,2,3"
-stage=0 # start from 0 if you need to start from data preparation
-stop_stage=6
+#export CUDA_VISIBLE_DEVICES="0"
+export CUDA_VISIBLE_DEVICES="6,7"
+stage=1 # start from 0 if you need to start from data preparation
+stop_stage=1
 # data
-data=/export/data/asr-data/OpenSLR/33/
+data=/export/expts4/chaoyang/
 data_url=www.openslr.org/resources/33
 
 nj=16
-feat_dir=fbank_pitch
+feat_dir=raw_wav
 dict=data/dict/lang_char.txt
 
-train_set=train_sp
+train_set=train
 # Optional train_config
 # 1. conf/train_transformer.yaml: Standard transformer
 # 2. conf/train_conformer.yaml: Standard conformer
 # 3. conf/train_unified_conformer.yaml: Unified dynamic chunk causal conformer
 train_config=conf/train_conformer.yaml
-checkpoint=
 cmvn=true
-dir=exp/sp_spec_aug
+dir=exp/conformer_cmvn
+#checkpoint=$dir/20.pt
```

Comment:
uncomment this, and keep checkpoint an empty variable

Reply:
Done.